src/flash/display/Loader.ts

```diff
@@ -350,7 +357,17 @@ module Shumway.AVM2.AS.flash.display {
       if (!release && traceLoaderOption.value) {
         console.log("Loading url " + request.url);
       }
-      this._fileLoader.loadFile(request._toFileRequest());
+      // Starting loading on the next script execution turn, in case allowDomain
```

Comment:
I hadn't even considered the option of just delaying the load itself. That certainly works, and is probably the easiest thing to do for now, nice.